CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+### citus v7.0.1 (September 12, 2017) ###
+
+* Fixes a bug causing memory leak in `INSERT ... SELECT` queries
```

Comment:
I think I'd prefer _that caused_ rather than _causing_, since the bug no longer exists.